pkg/cmd/builder/build_test.go

```diff
@@ -213,6 +216,15 @@ func TestParseBuildctlArgsForOCILayout(t *testing.T) {
 		},
 	}
 
+	if runtime.GOOS == "windows" {
+		abspath, err := filepath.Abs("/tmp/oci-layout")
```

Comment:
Shouldn't this be os.Getenv("TEMP")

Reply:
The path `/tmp/oci-layout` was / is hardcoded in the oci query part of the test:

https://github.com/containerd/nerdctl/blob/ca76611c84c34e606c8e4ef19047808573193906/pkg/cmd/builder/build_test.go#L206-L213

We could change it in both the test declaration and the assert - to GetTemp / Getenv (or whatever) else - but I am not sure it would serve much purpose: it seems this test is solely testing the parsing part of `parseBuildContextFromOCILayout` - which does exactly that (`filepath.Abs`) after stripping the scheme and will immediately error since there is no oci data in that location.

https://github.com/containerd/nerdctl/blob/ca76611c84c34e606c8e4ef19047808573193906/pkg/cmd/builder/build.go#L562

The path could very much be `/foo/bar` for all intent and purpose, and I presume who wrote the test did just put `/tmp/oci-layout` because that was the first thing they had in mind.

Lmk your thoughts.

Reply:
To be completely honest, I am not sure this test is bringing much (or any) value, but then, it is here - so...